src/DistributedWriteAheadLog/KafkaWALConsumer.cpp

```diff
@@ -75,7 +75,7 @@ void KafkaWALConsumer::initHandle()
         std::make_pair("bootstrap.servers", settings->brokers.c_str()),
         std::make_pair("group.id", settings->group_id),
         /// Enable auto offset commit
-        std::make_pair("enable.auto.commit", "true"),
+        std::make_pair("enable.auto.commit", std::to_string(settings->enable_auto_commit)),
```

Comment:
we shall never make this as `false`, we can remove this flag in settings

Reply:
fixed